src/DetailsView/components/base-left-nav.scss

```diff
@@ -5,9 +5,6 @@
 .details-view-test-nav-area {
     box-sizing: border-box;
     overflow-x: hidden;
-    max-height: calc(
```

Comment:
This css is not actually being applied because it's not valid. (the 2nd variable is not written correctly).